README.md

```diff
@@ -113,8 +119,18 @@ You can configure this template by passing different parameters in the Generator
 
 The most straightforward command to use this template is:
 ```bash
-ag https://bit.ly/asyncapi @asyncapi/nodejs-template -o output -p server=production
+$ ag https://bit.ly/asyncapi @asyncapi/nodejs-template -o output -p server=production
+```
+
+**Setup locally**
+
+```bash
+# Run following commands in terminal:
+$ git clone https://github.com/{username}/nodejs-template
+$ cd nodejs-template
+$ npm install
 ```
+**Note:** To test your changes locally refer [this](https://github.com/asyncapi/nodejs-template#cli).
```

Comment:
sorry, one last
maybe remove note and just add `ag https://bit.ly/asyncapi ./ -o output -p server=production. I know it is duplication but inside the same document, so yeah not a big deal I think

Reply:
No problem, I have made changes as you suggested 😊